### DISCUSSION TITLE
🚀 Feature:"AppWrite Crashlytics" sounds great!

### DISCUSSION THREAD

**Discussion by AlimFreight** [appwrite/appwrite#11512] [Ideas]
### 🔖 Feature description

I just wonder wouldn't be it a good idea to create the AppWrite Crashlytics ? This way it would be possible to avoid firebase completely. And it will be more efficiently as well. 

### 🎤 Pitch

I my Flutter app which is cross platform I have to use different solutions for different platforms because firebase doesn't support Windows and Linux for instance. So I would like to use one solution for any platform.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@AlimFreight, if you're looking for an alternative to Crashlytics, I think Sentry would be a good option. You can [self host](https://develop.sentry.dev/self-hosted/) it as well. We also have an integration where you can send error logs from Appwrite to sentry.
